packages/solid/utils/withScrolling.ts

```diff
@@ -117,14 +117,14 @@ function updateLastIndex(isRow: boolean, items: ElementNode): [{ position: numbe
         position: lastChild.x ?? 0,
         size: lastChild.width ?? 0
       },
-      items.width ?? 0
+      (items.preFlexwidth || items.width) ?? 0
```

Comment:
if `preFlexwidth` is 0 do we want it to use that value or pass on to `width`? we've usually been defaulting to taking the 0 in instances like that, but in this case it'd take the `width`

Reply:
Definitely not 0 - any width would be better. The renderer is now not redrawing nodes that have no size. So you want the row / column to have a size > the items inside of it.